packages/components/timeline/src/TimelineGroup.tsx

```diff
@@ -0,0 +1,185 @@
+import {
+  TimelineGroupProps,
+  TimelineProps,
+  TimelineGroupStyledProps,
+} from './timeline.types';
+import {
+  TimelineGroupCircleStyled,
+  TimelineGroupContainerStyled,
+  TimelineGroupContentStyled,
+  TimelineGroupLineStyled,
+  TimelineGroupSideStyled,
+} from './timeline.styles';
+import { deepMerge, useTheme, useWindowDimensions } from '@douro-ui/react';
+import { ReactNode } from 'react';
+import Timeline from './Timeline';
+
+const TimelineGroup = ({
+  options,
+  horizontally,
+  styled,
+  ...props
+}: TimelineGroupProps): ReactNode => {
+  const theme = useTheme();
+  const { width } = useWindowDimensions();
+  const isMobile = width !== undefined && width <= theme.breakpoints.tablet;
+
+  const defaultThemeValues: TimelineGroupStyledProps = {
+    backgroundColor: theme.colors.brand.white,
+    borderColor: theme.colors.neutral.silver.shade90,
+    borderColorHover: theme.colors.neutral.silver.shade50,
+    lineColor: theme.colors.extended.blue.shade90,
+    circleBackgroundColor: theme.colors.brand.white,
+    circleBorderColor: theme.colors.extended.blue.shade70,
+    circleBorderColorHover: theme.colors.extended.blue.shade40,
+  };
+
+  const mergedThemeValues = deepMerge<TimelineGroupStyledProps>(
+    defaultThemeValues,
+    styled,
+  );
+
+  const sortedOptions = [...options].sort(
```

Comment:
A more accurate name.

```suggestion
  const sortOptionsByDate = [...options].sort(
```